internal/util/vault.go

```diff
@@ -43,6 +43,9 @@ const (
 	// vault request headers
 	vaultTokenHeader     = "X-Vault-Token" // #nosec:G101, credential not vulnerable.
 	vaultNamespaceHeader = "X-Vault-Namespace"
+
+	// error code
+	errNotFound = 404
```

Comment:
this is a standard HTTP error code, use `http.StatusNotFound` instead, see https://golang.org/pkg/net/http/#pkg-constants

Reply:
Thanks. Will make the changes.